### PR TITLE
Add function to normalize HPC coordinates for Helioviewer

### PR DESCRIPTION
**Pull request by dgarciabriseno**

### CODE REVIEW

File: src/Types/HelioviewerEvent.php [hunk 1]
```diff
@@ -29,10 +29,16 @@ class HelioviewerEvent
...
     public array $views;
-    /** x position, use hgs2hpc if you only have a heliographic position */
-    public float $hpc_x;
-    /** y position, use hgs2hpc if you only have a heliographic position */
-    public float $hpc_y;
+    /**
+     * x position as it should appear on Helioviewer,
+     * use Coordinator::Hgs2Hpc if you only have a heliographic position
+     */
+    public float $hv_hpc_x;    // Reserved
```
Comment (dgarciabriseno) [inline src/Types/HelioviewerEvent.php]: Remove `Reserved`

File: src/Coordinator/Coordinator.php [hunk 2]
```diff
@@ -0,0 +1,53 @@
+<?php declare(strict_types=1);
+
+namespace HelioviewerEventInterface\Coordinator;
+
+use \Exception;
```
Comment (dgarciabriseno) [inline src/Coordinator/Coordinator.php:5]: Create Coordinator Exception

File: src/Coordinator/Coordinator.php [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+     */
+    public static function HPC(float $x, float $y, string $obstime) {
+        try {
+            $result = file_get_contents(HV_COORDINATOR_URL . "/hpc?x=" . $x . "&y=" . $y . "&obstime=" . urlencode($obstime));
+            $response = json_decode($result, true);
+            // The response format from the API is just x, y, but better to
+            // not make any assumptions. Handle the error here instead of
+            // on the client.
+            return array("x" => $response["x"], "y" => $response["y"]);
+        } catch (Exception $e) {
```
Comment (dgarciabriseno) [inline src/Coordinator/Coordinator.php]: Don't catch all exceptions

File: tests/translators/FlarePredictionTest.php [hunk 4]
```diff
@@ -18,8 +18,8 @@ public function testCoordinates(): void
             foreach ($section['groups'] as $group) {
                 foreach ($group['data'] as $record) {
                     $count += 1;
-                    $this->assertNotEquals(0.123456789, $record['hpc_x']);
-                    $this->assertNotEquals(0.987654321, $record['hpc_y']);
+                    isset($record['hpc_x']);
```
Comment (mudhoney) [inline tests/translators/FlarePredictionTest.php]:
```
 $this->assertArrayHasKey('foo', ['bar' => 'baz']);
 ```
 ^ might be usefull for testing these

**Review by mudhoney — APPROVED** [review]

File: tests/translators/FlarePredictionTest.php (hunk 4 above)
Comment (dgarciabriseno) [reply tests/translators/FlarePredictionTest.php]: Updated

**Review by mudhoney — APPROVED** [review]
All good!!

**Pull request merged**